### PR TITLE
fix(docs): fix README webpack syntax error

### PR DESCRIPTION
**Pull request by codymikol**
make the webpack devtool configuration in README
  syntactically correct and consistent with the
  other configurations in the file.
  
  Fixes N/A

This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [x] **typo fix**
- [ ] **metadata update**

### Motivation / Use-Case

Hopefully this will save someone a fraction of the second whilst shamelessly copy and pasting the devtool configuration into their own project 😉

### Breaking Changes

Nope

### Additional Info

Nope

### CODE REVIEW

**Comment by codymikol** [comment]
Signed

**Comment by codymikol** [comment]
At this point it seems to be failing because of an NPM audit, the linter actually passes just fine

File: README.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ And tell `webpack` to generate sourcemaps.
 ```js
 webpack: {
   // ...
-  devtool: 'inline-source-map';
+  'devtool': 'inline-source-map'
```
Comment (Pezmc) [inline README.md:129]: The `''`'s shouldn't be needed here!

Comment (codymikol) [reply README.md:129]: This is needed because of the linter rules of this project, if you remove the single quotes the CI lint will fail.

**Pull request closed without merging**